release/serve_tests/workloads/autoscaling_load_test.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+"""
+Benchmark test.
+"""
+
+import logging
+
+import click
+from anyscale import service
+from anyscale_service_utils import start_service
+from locust_utils import LocustLoadTestConfig, LocustStage, run_locust_load_test
+from serve_test_utils import save_test_results
+
+import ray
+
+
+logger = logging.getLogger(__file__)
+logging.basicConfig(level=logging.INFO)
+
+
+URI = "https://serve-resnet-benchmark-data.s3.us-west-1.amazonaws.com/000000000019.jpeg"
+
+
+@click.command()
+def main():
+    resnet_application = {
+        "name": "default",
+        "import_path": "resnet_50:app",
+        "route_prefix": "/",
```

Comment:
nit: drop unnecessary fields
```suggestion
        "import_path": "resnet_50:app",
```